docs/azure/landing-page.yml

```
@@ -9,7 +9,7 @@ metadata:
   ms.topic: landing-page
   ms.collection: collection
   ms.technology: azure
-  ms.service: multiple
+  ms.service: app-service
```

```suggestion
  ms.service: azure
```

I believe that `ms.service: azure` is more appropriate. This content is not limited to Azure App Service, but covers much more Azure offerings. See https://review.docs.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#msservice

---

Hi @IEvangelist, `ms.service: azure` is scheduled for removal from the allowlist. You can contact the Learn Platform Metadata ([learn-metadata@microsoft.com](mailto:learn-metadata@microsoft.com)) for more information regarding the allowlist. In the meantime, I'd be happy to change this value to another approved one that is better suited for this content!

---

Hi @brittmsantos - what about `cloud-adoption-framework`. That seems to cover a lot of the concepts driving this area of the content. It's all about how .NET can intise the adoption of Azure, no?

---

That works for me! I've gone ahead and implemented this change.